### ISSUE TITLE
"Unroll repeats" vs "Expand repeats"

### ISSUE THREAD

**Issue by Jojo-Schmitz** [musescore/MuseScore#19649]
### Issue type

UI bug

### Steps to reproduce

Seems an inconsistent naming for basically the same thing, one in Tools, the other in Export MIDI

### MuseScore Version

4.1.1

### Regression

No.

### Operating system

Windows 11

**Comment by Jojo-Schmitz** [comment]
See also #9670

**Comment by wizofaus** [comment]
That issue is locked too, and I'd happy to put my hand to finish off what I'd started with it (well, I'll have to start again as I lost the branch...)

**Comment by wizofaus** [comment]
However I don't really understand the point of this particular issue - my question would be why is "expand repeats" even an option for export MIDI, it isn't for export mp3 etc. - but I accept that you may simply wish to use MIDI as a format to capture *just* the semantic info around notes (pitches/durations etc.) and anything else will have to be re-done by hand in whatever other tool you're importing into.  At any rate, the "unroll repeats" command is obviously different, I don't see a huge issue with using a different verb.

**Comment by Jojo-Schmitz** [comment]
Yes, I believe expand repeats should be an option for all audio formats, but is esp. needed for MIDI, to communicate to MIDI devices
